<<<<<<<< HEAD:patches/monero/0003-uint64_t-missing-definition-fix.patch
From a74f616e3c1671a883182b0db1c6fc11a1242c01 Mon Sep 17 00:00:00 2001
From: Czarek Nakamoto <cyjan@mrcyjanek.net>
Date: Mon, 2 Sep 2024 16:40:31 +0200
Subject: [PATCH 03/14] uint64_t missing definition fix
========
From 223e2e4b5b2c6366d0dd4130a726c4cfcdb13ffb Mon Sep 17 00:00:00 2001
From: Czarek Nakamoto <cyjan@mrcyjanek.net>
Date: Mon, 2 Sep 2024 16:40:31 +0200
Subject: [PATCH 05/15] uint64_t missing definition fix
>>>>>>>> e3d763b (cleanup patches (and other stuff) (#79)):patches/wownero/0005-uint64_t-missing-definition-fix.patch

---
 contrib/epee/include/net/http_base.h | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/contrib/epee/include/net/http_base.h b/contrib/epee/include/net/http_base.h
index 4af4da790..ae4c0d05e 100644
--- a/contrib/epee/include/net/http_base.h
+++ b/contrib/epee/include/net/http_base.h
@@ -28,7 +28,7 @@
 
 #pragma once
 #include "memwipe.h"
-
+#include <stdint.h>
 #include <boost/utility/string_ref.hpp>
 
 #include <string>
-- 
<<<<<<<< HEAD:patches/monero/0003-uint64_t-missing-definition-fix.patch
2.48.1
========
2.39.5 (Apple Git-154)
>>>>>>>> e3d763b (cleanup patches (and other stuff) (#79)):patches/wownero/0005-uint64_t-missing-definition-fix.patch

